CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## [v1.0.58] _2021-02-02_
+## [v1.0.59] _2021-02-05_
```

Comment:
It should be `1.0.61` no?

Reply:
https://github.com/cycloidio/cycloid-cli/pull/91